website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -1,500 +1,110 @@
-	...
-)
-
-func (r *MemcachedReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-  // Lookup the Memcached instance for this reconcile request
-  memcached := &cachev1alpha1.Memcached{}
-  err := r.Get(ctx, req.NamespacedName, memcached)
-  ...
-}
-```
+### Configuring your test environment
 
-Based on the return values, [`Result`][result_go_doc] and error, the `Request` may be requeued and the reconcile loop may be triggered again:
-
-```Go
-// Reconcile successful - don't requeue
-return ctrl.Result{}, nil
-// Reconcile failed due to error - requeue
-return ctrl.Result{}, err
-// Requeue for any reason other than an error
-return ctrl.Result{Requeue: true}, nil
-```
-
-You can set the `Result.RequeueAfter` to requeue the `Request` after a grace period as well:
-```Go
-import "time"
-
-// Reconcile for any reason other than an error after 5 seconds
-return ctrl.Result{RequeueAfter: time.Second*5}, nil
-```
-
-**Note:** Returning `Result` with `RequeueAfter` set is how you can periodically reconcile a CR.
-
-For a guide on Reconcilers, Clients, and interacting with resource Events, see the [Client API doc][doc_client_api].
-
-### Specify permissions and generate RBAC manifests
-
-The controller needs certain RBAC permissions to interact with the resources it manages. These are specified via [RBAC markers][rbac_markers] like the following:
-
-```Go
-// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/status,verbs=get;update;patch
-// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=core,resources=pods,verbs=get;list;
-
-func (r *MemcachedReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-```
-
-The `ClusterRole` manifest at `config/rbac/role.yaml` is generated from the above markers via controller-gen with the following command:
-
-```sh
-$ make manifests
-```
-
-## Build and run the operator
-
-Before running the operator, the CRD must be registered with the Kubernetes apiserver:
-
-```sh
-$ make install
-```
-
-Once this is done, there are two ways to run the operator:
-
-- As Go program outside a cluster
-- As a Deployment inside a Kubernetes cluster
-
-## Configuring your test environment
-
-Projects are scaffolded with unit tests that utilize the [envtest](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/envtest)
-library, which requires certain Kubernetes server binaries be present locally.
-Installation instructions can be found [here][env-test-setup].
-
-### 1. Run locally outside the cluster
-
-To run the operator locally execute the following command:
+Projects are scaffolded with tests that utilize the [`envtest`][env-test]
+library, which requires certain Kubernetes server binaries to be present locally. Update your Makefile by replacing your `test` target with: 
```

Comment:
Most of this is captured in the envtest doc. Let's link to the doc in the first sentence, and then have a single imperative action for the user to do:

```suggestion
[Setup the `envtest` binaries and environment][env-test-setup] for your project.

Update your `test` Makefile target to the following:
```